engine/src/main/java/org/terasology/config/flexible/AutoConfigManager.java

```diff
@@ -48,6 +50,7 @@
 
     public AutoConfigManager(TypeHandlerLibrary typeHandlerLibrary) {
         this.typeHandlerLibrary = typeHandlerLibrary;
+        typeHandlerLibrary.addTypeHandler(Locale.class, new LocaleTypeHandler());
```

Comment:
Since `Locale` is a SDK class, can we add this type handler by default to every type handler library (and hence move it into one of the initialization methods in `TypeHandlerLibrary`)?